src/panels/lovelace/editor/select-view/hui-dialog-select-view.ts

```diff
@@ -94,12 +96,43 @@ export class HuiDialogSelectView extends LitElement {
             </ha-paper-dropdown-menu>`
           : ""}
         ${this._config
-          ? html` <hui-views-list
```

Comment:
This is the only place `hui-views-list` was used, it is no longer used now, so should be removed.